src/styles/components/_button.scss

```
@@ -1,18 +1,128 @@
 .button {
   cursor: pointer;
-  height: 40px;
-  width: 100%;
-  background-color: var(--gray-1);
+  border: none;
+  background-color: var(--white);
   font: var(--button);
+}
+
+.button--large {
+  height: 40px;
+  width: 295px;
```

Aquil ele poderia expandir, fica mais responsivo. Onde vamos usar ele também é somente ele na "linha"
```suggestion
  width: 100%;
```